src/Hive.Plugins/Aggregates/AggregatedInstanceGenerator.cs

```diff
@@ -8,6 +8,7 @@
 using System.Text;
 using Hive.Plugins.Aggregates;
 using Hive.Plugins.Resources;
+using Microsoft.Extensions.DependencyInjection;
```

Comment:
Is this the correct include, especially now that we are using DryIoc?

Reply:
I said include, I meant using... My brain has been C-ified

Reply:
Yes, because it needs to use `ActivatorUtilities` in that namespace, and Hive.Plugins already references that nuget package, and I'd rather not make it depend on DryIoc for just this one thing, and be locked in.